site2/docs/functions-runtime.md

```diff
@@ -278,7 +278,11 @@ The Kubernetes integration enables you to implement a class and customize how to
 
 The functions (and sinks/sources) API provides a flag, `customRuntimeOptions`, which is passed to this interface.
 
-Pulsar includes a built-in implementation. To use the basic implementation, set `runtimeCustomizerClassName` to `org.apache.pulsar.functions.runtime.kubernetes.BasicKubernetesManifestCustomizer`. The built-in implementation enables you to pass a JSON document with certain properties to augment how the manifests are generated. The following is an example.
+To initialize the `KubernetesManifestCustomizer`, you can provide `runtimeCustomizerConfig` in the `functions-worker.yml` file. `runtimeCustomizerConfig` is passed to the `public void initialize(Map<String, Object> config)` function of the interface. `runtimeCustomizerConfig`is different from the `customRuntimeOptions` as `runtimeCustomizerConfig` is the same across all functions. If you provide both `runtimeCustomizerConfig`  and `customRuntimeOptions`, you need to decide how to manage these two configurations in your implementation of `KubernetesManifestCustomizer`. 
+
+Pulsar includes a built-in implementation. To use the basic implementation, set `runtimeCustomizerClassName` to `org.apache.pulsar.functions.runtime.kubernetes.BasicKubernetesManifestCustomizer`. The built-in implementation initialized with `runtimeCustomizerConfig` defined in the `functions-worker.yml` file enables you pass a JSON document as `customRuntimeOptions` with certain properties to augment to decide how the manifests are generated. If both `runtimeCustomizerConfig`  and `customRuntimeOptions` are provided, `BasicKubernetesManifestCustomizer` uses `customRuntimeOptions` to override the configuration if there are conflicts in these two configurations. 
```

Comment:
The built-in implementation initialized with `runtimeCustomizerConfig` defined in the `functions-worker.yml` file enables you pass a JSON document as `customRuntimeOptions` with certain properties to augment to decide how the manifests are generated.
For easy of read, it's better to break these long sentence into several short sentences.

Reply:
1. I have the same concern on the long sentences.
2. "enables you pass..." --> "enables you to pass..."

Reply:
@Huanli-Meng @Jennifer88huang thanks for your comments. Updated, PTAL.